x-pack/plugins/ml/public/application/explorer/components/explorer_query_bar/explorer_query_bar.tsx

```diff
@@ -127,30 +135,50 @@ export const ExplorerQueryBar: FC<ExplorerQueryBarProps> = ({
     setSearchInput(query);
   };
   const applyInfluencersFilterQuery = (query: Query) => {
-    const { clearSettings, settings } = getKqlQueryValues({
-      inputString: query.query,
-      queryLanguage: query.language,
-      indexPattern,
-    });
+    try {
+      const { clearSettings, settings } = getKqlQueryValues({
+        inputString: query.query,
+        queryLanguage: query.language,
+        indexPattern,
+      });
 
-    if (clearSettings === true) {
-      explorerService.clearInfluencerFilterSettings();
-    } else {
-      explorerService.setInfluencerFilterSettings(settings);
+      if (clearSettings === true) {
+        explorerService.clearInfluencerFilterSettings();
+      } else {
+        explorerService.setInfluencerFilterSettings(settings);
+      }
+    } catch (e) {
+      console.log('Invalid query syntax in search bar', e); // eslint-disable-line no-console
+      setErrorMessage({ query: query.query as string, message: e.message });
     }
   };
 
   return (
-    <QueryStringInput
-      bubbleSubmitEvent
-      query={searchInput}
-      indexPatterns={[indexPattern]}
-      onChange={searchChangeHandler}
-      onSubmit={applyInfluencersFilterQuery}
-      placeholder={filterPlaceHolder}
-      disableAutoFocus
-      dataTestSubj="explorerQueryInput"
-      languageSwitcherPopoverAnchorPosition="rightDown"
-    />
+    <EuiInputPopover
+      style={{ maxWidth: '100%' }}
+      closePopover={() => setErrorMessage(undefined)}
+      input={
+        <QueryStringInput
+          bubbleSubmitEvent
+          query={searchInput}
+          indexPatterns={[indexPattern]}
+          onChange={searchChangeHandler}
+          onSubmit={applyInfluencersFilterQuery}
+          placeholder={filterPlaceHolder}
+          disableAutoFocus
+          dataTestSubj="explorerQueryInput"
+          languageSwitcherPopoverAnchorPosition="rightDown"
+        />
+      }
+      isOpen={errorMessage?.query === searchInput.query && errorMessage?.message !== ''}
+    >
+      <EuiCode>
+        {i18n.translate('xpack.ml.explorer.invalidKuerySyntaxErrorMessageQueryBar', {
+          defaultMessage: 'Invalid Query',
```

Comment:
Should use sentence case here `Invalid query`

Reply:
Updated cfd728fb86eabd3b00697a735c2fa9293d487db3